main.js

```diff
@@ -318,20 +318,34 @@ function storeCurrentActivity() {
     loggedActivities = JSON.parse(localStorage.getItem('activitiesArray'));
     loggedActivities.unshift(currentActivity);
     displayLoggedActivities();
+    hideTimer();
     activitiesArray = JSON.stringify(loggedActivities)
     localStorage.removeItem('activitiesArray');
     localStorage.setItem('activitiesArray', activitiesArray)
   } else {
     loggedActivities.unshift(currentActivity)
     displayLoggedActivities();
+    hideTimer();
     activitiesArray = JSON.stringify(loggedActivities)
     localStorage.setItem('activitiesArray', activitiesArray)
   }
 }
 
+window.addEventListener('load', displayOnLoad)
+
+function displayOnLoad() {
+  if (localStorage.getItem('activitiesArray')) {
+    hide(haventLoggedMessage)
+    hide(completeFormMessage)
+    loggedActivities = JSON.parse(localStorage.getItem('activitiesArray'));
+    displayLoggedActivities()
+  }
+}
```

Comment:
I like how you moved the hideTimer() so that it's happening at the log activity button